scripts/pre_redeploy.sh

```diff
@@ -106,8 +106,16 @@ pushd ${LEAPFROG_DIR}
 popd
 
 if [[ ! -f "${UPGRADE_LEAP_MARKER_FOLDER}/configure-apt-sources-rpc.complete" ]]; then
-   pushd ${RPCO_DEFAULT_FOLDER}/rpcd/playbooks
-       openstack-ansible configure-apt-sources.yml
-   popd
-   log "configure-apt-sources-rpc" "ok"
+    pushd ${RPCO_DEFAULT_FOLDER}/rpcd/playbooks
+        openstack-ansible configure-apt-sources.yml
+    popd
+    log "configure-apt-sources-rpc" "ok"
 fi
+
+pushd /etc/openstack_deploy/
```

Comment:
Made a few changes and tested this out, seems to work:

```
pushd /etc/openstack_deploy/
    if ! grep "openstack_domain" user_osa_variables_defaults.yml > /dev/null; then
        echo "openstack_domain: ''" >> user_osa_variables_defaults.yml
    else
        sed -i s/^openstack_domain.*/openstack_domain:\ \'\'/g user_osa_variables_defaults.yml
    fi
popd
```
The existing one had a few typos on the file name and would also add openstack_domain to the end of the file each time it ran.